docs/test_cases/TestSuite42.md

```diff
@@ -0,0 +1,194 @@
+# Test Suite 42
+
+**Business Term**
+
+## Test Case 42.1
+**View or Edit Business Term**
+
+### Test Assertion:
+
+#### Test Assertion #42.1.1
+The end user can create a Business Term with all information specified. 
+
+#### Test Assertion #42.1.2
+The end user can search for a Business Term based only on its term. 
+
+#### Test Assertion #42.1.3
+The end user can search for a Business Term based on the external reference URI.
+
+#### Test Assertion #42.1.4
+The end user can select a Business Term in the table to update its details in Edit Business Term page. 
+
+#### Test Assertion #42.1.5
+The end user cannot change Definition field in Edit Business Term page. 
```

Comment:
Why the end user can't change the definition?

Reply:
According to the Design,
 
Update is available when at least one attribute is changed). Please note that the definition can not be changed using UI, it can be just imported from the file.
 
https://oagiscore.atlassian.net/wiki/spaces/SCORE/pages/4312137769/Business+Term+Implementation+Use+Cases+and+Design+Views

Reply:
My concern is that the user who uploaded the template with the definition should upload it again to update the definition.